packages/feedback-spa/README.md

```diff
@@ -1,23 +1,57 @@
-# Feedback
+# Feedback SPA
+Feedback SPA provides the consolidated view of user experience which has shared to developer team.
```

Comment:
Can you update readme with a short description?

Reply:
updated

Reply:
Also add a newline after the heading.

Reply:
Added